pkgs/development/tools/treefmt/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+rustPlatform.buildRustPackage rec {
```

Comment:
```suggestion
{ lib, rustPlatform, fetchFromGitHub }:

rustPlatform.buildRustPackage rec {
```

Reply:
I don't see the purpose of this one. What advantage is there to have an additional empty line?

Reply:
It clearly sepearates inputs from the actual package.

Reply:
Syntax highlighting already does this for me. I think this is more a matter of habit?